pkg/apis/networking/v1/defaults_test.go

```diff
@@ -234,6 +237,119 @@ func TestSetDefaultNetworkPolicy(t *testing.T) {
 	}
 }
 
+func TestSetDefaultIngressClassEnabled(t *testing.T) {
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate,
```

Comment:
You can make this part of your test case so you don't need 2 different test functions: https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/networking/networkpolicy/strategy_test.go#L200.